usaspending_api/references/v2/views/autocomplete.py

```diff
@@ -152,21 +152,23 @@ def post(self, request):
         search_text, limit = self.get_request_payload(request)
 
         # get relevant TransactionFPDS
-        queryset = TransactionFPDS.objects.filter(naics__isnull=False, naics_description__isnull=False)
+        queryset = NAICS.objects.all()
         # Filter based on search text
         response = {}
 
         queryset = queryset.annotate(similarity=Greatest(
-            TrigramSimilarity('naics', search_text),
-            TrigramSimilarity('naics_description', search_text)))\
+            TrigramSimilarity('code', search_text),
+            TrigramSimilarity('description', search_text)))\
             .distinct().order_by('-similarity')
 
         naics_exact_match_queryset = queryset.filter(similarity=1.0)
         if naics_exact_match_queryset.count() > 0:
             queryset = naics_exact_match_queryset
 
-        results_set = list(queryset.values('naics', 'naics_description')[:limit]) if limit else list(
-            queryset.values('naics', 'naics_description'))
+        results_set = list(queryset.values('code', 'description')[:limit]) if limit else list(
```

Comment:
Can we use `F` to alias the columns from `code` to `naics` and same for description? would that get ride of the need for the list comprehension below?